src/mediaplayer.js

```diff
@@ -1944,6 +1951,30 @@ const mediaplayer = {
         } else if (this.autoStart) {
             this.play();
         }
+
+        if(this.$container && this.config.height && this.config.height !== 'auto') {
+            this._setMaxHeight();
+        }
+    },
+
+    /**
+     * Set max height limit for container
+     * using by old media items with defined height.
+     * @private
+     */
+    _setMaxHeight: function _setMaxHeight() {
+        const $video = this.$container.find('video.video');
```

Comment:
This check should be only for video (not youtube). And use `videoWidth` and `videoHeight` see example: 
https://github.com/oat-sa/tao-core-ui-fe/blob/4c85f2a4416124ea82d310ff01b4697f5a47b77b/src/mediaplayer.js#L1448-L1460

Reply:
Thanks for the review.
1) why not youtube? (but it is skipping all new items with height: 'auto')
2) this is different, maybe I should rename properties. Here I using the box sizes of the video container.
Basically, I scale down the width of the container to make sure it's fitting to the max height.
Without that width will be pushed to 100%.

Reply:
Thanks!